src/python/TaskWorker/Actions/PostJob.py

```diff
@@ -1513,6 +1517,7 @@ def execute(self, *args, **kw):
             self.logger.error(msg)
             if self.crab_retry is not None:
                 self.set_dashboard_state('FAILED')
+                self.set_postJob_ClassAds('FAILED')
```

Comment:
I'd rather use a more descriptive name for this method, like self.report_state or self.send_state_to_monitor, or even self.set_state_classAd if we want to have an implementation dependent name

Reply:
maybe `set_ClassAds_value`?

Reply:
there are tons of ads areound, please have a name which makes it clear that only set one or two (at least IIUC).